internal/store/elasticsearch/es.go

```diff
@@ -65,6 +60,43 @@ type searchResponse struct {
 	} `json:"aggregations"`
 }
 
+type groupResponse struct {
+	Aggregations struct {
+		TermAggregations TermAggregations `json:"term_agg"`
+	} `json:"aggregations"`
+}
+
+type aggregationBucket struct {
+	Key      string `json:"key"`
+	DocCount int    `json:"doc_count"`
+	Hits     struct {
+		Hits struct {
+			Hits []groupHits `json:"hits"`
+		} `json:"hits"`
+	} `json:"hits"`
+}
+
+type groupHits struct {
+	Source asset.Asset `json:"_source"`
+}
+
+type TermAggregations struct {
+	DocCountErrorUpperBound int `json:"doc_count_error_upper_bound"`
+	SumOtherDocCount        int `json:"sum_other_doc_count"`
+	Buckets                 []Buckets
+}
+
+type Buckets struct {
+	Key      string
```

Comment:
Is the `json` tag explicitly excluded here ?

Reply:
it was not required explicitly hence skipped. It automatically maps to "key"